src/arr.ts

```diff
@@ -37,6 +46,61 @@ export class Arr<T> {
     return this.filter(Boolean)
   }
 
+  /**
+   * Breaks the array into multiple, smaller arrays
+   * of the given `size`.
+   *
+   * @param {Number} size
+   *
+   * @returns {Array}
+   */
+  chunk (size: number): any[] {
+    const chunks = []
+
+    while (this.size()) {
+      chunks.push(
+        this.values.splice(0, size)
+      )
+    }
+
+    return chunks
+  }
+
+  /**
+   * Removes all values from the array that are present in the given array.
+   *
+   * @param {*} values
+   *
+   * @returns {Arr}
+   */
+  diff (values: any[]): Arr<T> {

```

Comment:
Let’s use types to improve the developer experience

```suggestion
  diff (values: T[]): Arr<T> {
```